collab-service/src/message_handler/room/connect_message_builder.ts

```diff
@@ -0,0 +1,24 @@
+import * as encoding from 'lib0/encoding';
+
+export const OPCODE_USER_JOIN = 4;
+export const OPCODE_USER_LEAVE = 5;
+
+function createConnectionMessage(username: string, opcode: number): Uint8Array {
```

Comment:
Naming can be improved here.
```suggestion
function createConnectionAlertMessage(username: string, opcode: number): Uint8Array {
```